src/Common/ErrorCodes.cpp

```diff
@@ -555,6 +555,7 @@
     M(585, CANNOT_PARSE_YAML) \
     M(586, CANNOT_CREATE_FILE) \
     M(587, CONCURRENT_ACCESS_NOT_SUPPORTED) \
+    M(588, DISTINCT_ON_AND_LIMIT_BY_TOGETHER) \
```

Comment:
I see that we have separate specific error codes for similar cases. But maybe better to reuse some to not introduce new one, but I'm not sure.

Reply:
But I can't find this error code. Can u please help me figure it out?

Reply:
`62` doesn't look suitable, actually I want to find suitable existing code, but seems there isn't. Ok, let's leave as is, I'll change it if I'll find solution or leave current one.

Reply:
I've changed to `UNSUPPORTED_METHOD`